### PR TITLE
Runner - Handle throwables too

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by keradus** [comment]
Please rebase @GrahamCampbell , I would like to merge it

**Comment by GrahamCampbell** [comment]
@keradus Done. :)

**Comment by keradus** [comment]
hmmmm
> This branch is out-of-date with the base branch

**Comment by keradus** [comment]
What about incorporating #1858 here ?

**Comment by GrahamCampbell** [comment]
This PR is to 2.x though, not 1.x.

File: src/Runner/Runner.php [hunk 1]
```diff
@@ -195,14 +195,9 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
...
-                FixerFileProcessedEvent::NAME,
-                FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_EXCEPTION)
-            );
-
-            $this->errorsManager->report(new Error(Error::TYPE_EXCEPTION, $name));
-
-            return;
+            return $this->processException($name);
+        } catch (\Throwable $e) {
+            return $this->processException($name);
```
Comment (keradus) [inline src/Runner/Runner.php]: processException is void, we shouldn't return it

Comment (GrahamCampbell) [inline src/Runner/Runner.php]: that's ok, that's the same as just `return`

Comment (GrahamCampbell) [inline src/Runner/Runner.php]: it was just nice cs wise to keep it on the same line

Comment (keradus) [inline src/Runner/Runner.php]: It may not be ok when we strictly typehint that and then grab output of it.

Comment (GrahamCampbell) [inline src/Runner/Runner.php]: Ok, I'll change it.

**Comment by keradus** [comment]
Thank you @GrahamCampbell.

**Comment by GrahamCampbell** [comment]
:)

**Pull request merged**